tests/test_4520.py

```diff
@@ -0,0 +1,19 @@
+import pymupdf
+
+
+def test_4520():
+    """Accept source pages without /Contents object in show_pdf_page."""
+    vsn_tuple = tuple(map(int, pymupdf.__version__.split(".")))
```

Comment:
You could use `pymupdf.pymupdf_version_tuple` here.

However we don't generally try to make tests work with different versions of PyMuPDF. For example there will be test failures if one does `pip install pymupdf==1.25.5; pytest PyMuPDF`.

It's different with mupdf versions where we often look at `pymupdf.mupdf_version_tuple` so we can test the current PyMuPDF with different versions of MuPDF (generally the current release branch (1.26.x) and master).